Appraisals

```diff
@@ -1,15 +1,19 @@
 appraise "rails-6.0" do
   gem 'rails', '~> 6.0'
+  gem "rspec-rails", '~> 5'
```

Comment:
[chore(dep): modify rspec version for each rails ver](https://github.com/Coursemology/activerecord-userstamp/pull/4/commits/5de7cc00f9fd24d505d97ada870600175b357248)
should squash with  
first commit [chore(test): setup test and change requirement](https://github.com/Coursemology/activerecord-userstamp/pull/4/commits/db72db7cc2ab9fb6172e59a17ea85bae98ac1944)

Clean these up?

Reply:
done